cypress/integration/console/devices/edit.spec.js

```diff
@@ -117,6 +119,76 @@ describe('Device general settings', () => {
       .should('be.visible')
   })
 
+  it('succeeds editing Network layer', () => {
+    const device = {
+      dev_addr: generateHexValue(8),
+      nwk_s_key: generateHexValue(32),
+      lorawan_version: 'MAC_V1_0',
+      frequency_plan_id: '863-870 MHz',
+    }
+
+    cy.findByText('Network layer', { selector: 'h3' })
+      .closest('[data-test-id="collapsible-section"]')
+      .within(() => {
+        cy.findByRole('button', { name: 'Expand' }).click()
+        cy.findByLabelText('Frequency plan').type(device.frequency_plan_id)
+        cy.findByLabelText('LoRaWAN version').type(device.lorawan_version)
+        cy.findByLabelText('Device address').type(device.dev_addr)
+        cy.findByLabelText('NwkSKey').type(device.nwk_s_key)
+        cy.findByRole('button', { name: 'Save changes' }).click()
+      })
+
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.findByTestId('toast-notification')
+      .should('be.visible')
+      .findByText(`End device updated`)
+      .should('be.visible')
```

Comment:
```suggestion
    cy.findByTestId('toast-notification')
      .findByText(`End device updated`)
      .should('be.visible')
```

I believe checking the visibility here twice is redundant.